lib/Init.vala

```diff
@@ -70,6 +70,9 @@ namespace Granite {
             if (dark_provider == null) {
                 dark_provider = new Gtk.CssProvider ();
                 dark_provider.load_from_resource ("/io/elementary/granite/Granite-dark.css");
+                #if INCLUDE_GTK
+                    dark_provider.load_from_resource ("/io/elementary/granite/Gtk-dark.css");
+                #endif
```

Comment:
```suggestion
#if INCLUDE_GTK
                dark_provider.load_from_resource ("/io/elementary/granite/Gtk-dark.css");
#endif
```

Preprocessor `#if`s are different from normal `if` statement, so you shouldn't make indentation.